Current Master branch contains Golang linting issues

**Describe the Bug:**
`make check` fails golang lint checks.

**Steps to Reproduce:**
`make check`
Results:

./devtools/ci/tasks/scripts/codecheck.sh
Found non-formatted files:
./cmd/goalert-slack-email-sync/main.go
./config/config.go
./user/user.go
./permission/permission.go
make: *** [check] Error 1

**Expected Behavior:**
Expecting no errors.

**Observed Behavior:**
As described above.

**Screenshots/Stack Traces:**

**Application Version:**
master branch

**Desktop:**
NA

**Smartphone/Tablet:**
NA

**Additional Context:**

--- a/cmd/goalert-slack-email-sync/main.go
+++ b/cmd/goalert-slack-email-sync/main.go
@@ -1,5 +1,5 @@
 /*
-	goalert-slack-email-sync will create/update AuthSubject entries for users by matching the user's GoAlert email to the corresponding Slack user.
+goalert-slack-email-sync will create/update AuthSubject entries for users by matching the user's GoAlert email to the corresponding Slack user.
 */
 package main
 

--- a/config/config.go
+++ b/config/config.go
@@ -231,7 +231,7 @@ func (cfg Config) CallbackURL(path string, mergeParams ...url.Values) string {
 	return base.String()
 }
 
-//  MatchURL will compare two url strings and will return true if they match.
+// MatchURL will compare two url strings and will return true if they match.
 func MatchURL(baseURL, testURL string) (bool, error) {
 	compareQueryValues := func(baseVal, testVal url.Values) bool {
 		for name := range baseVal {

--- a/permission/permission.go
+++ b/permission/permission.go
@@ -1,5 +1,4 @@
 /*
-
 Package permission handles checking and granting of permissions using context.Context.
 
 A context can be granted User, System, or Service privileges using UserContext, SystemContext, or ServiceContext, respectively.
@@ -7,6 +6,5 @@ A context can be granted User, System, or Service privileges using UserContext, 
 Data can be extracted using the appropriate method (e.g. UserID, ServiceID, etc...)
 
 Context can then be validated using Checkers (e.g. like the User function) or by using LimitCheckAny and a number of Checkers together.
-
 */
 package permission

--- a/user/user.go
+++ b/user/user.go
@@ -14,7 +14,6 @@ import (
 
 // A User is the base information of a user of the system. Authentication details are stored
 // separately based on the auth provider.
-//
 type User struct {
 	// ID is the unique identifier for the user
 	ID string

